plugins/Lv2Effect/Lv2FxControls.h

```diff
@@ -60,8 +60,8 @@ private slots:
 	void changeControl();
 
 private:
-	DataFile::Types settingsType() override;
-	void setNameFromFile(const QString &name) override;
+	void onSampleRateChanged();
+	void reload();
```

Comment:
There's no definition for `Lv2FxControls::reload`, making macOS builds fail.

Reply:
Thanks, I did not see that. Fixed it now.